src/figuresClasses.ts

```diff
@@ -1,11 +1,87 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (a <= 0 || b <= 0 || c < +0) {
+      throw new Error('Length of sides must be greater than 0');
```

Comment:
[CODE STYLE] - The condition `c < +0` seems to be a typo. It should be `c <= 0` to maintain consistency with the other conditions.